src/App.tsx

```diff
@@ -1,11 +1,7 @@
-import "./App.css";
+import Home from "../src/pages/Home";
 
 function App() {
-  return (
-    <>
-      <h1>React Payments</h1>
-    </>
-  );
+  return <Home></Home>;
```

Comment:
사소한 내용이지만, 안에 children을 받지 않는 컴포넌트의 경우 아래처럼 명확히 단일 태그라는 것을 보여주면 좋아요.
```suggestion
  return <Home />;
```